src/ImmersedBoundaries/grid_fitted_bottom.jl

```diff
@@ -24,18 +24,18 @@ struct InterfaceImmersedCondition end
 Base.summary(::CenterImmersedCondition) = "CenterImmersedCondition"
 Base.summary(::InterfaceImmersedCondition) = "InterfaceImmersedCondition"
 
-"""
-    GridFittedBottom(bottom_height, [immersed_condition=CenterImmersedCondition()])
-
-Return an immersed boundary with an irregular bottom fit to the underlying grid.
-"""
 struct GridFittedBottom{H, I} <: AbstractGridFittedBottom{H}
     bottom_height :: H
     immersed_condition :: I
 end
 
 const GFBIBG = ImmersedBoundaryGrid{<:Any, <:Any, <:Any, <:Any, <:Any, <:GridFittedBottom}
 
+"""
+    GridFittedBottom(bottom_height, [immersed_condition=CenterImmersedCondition()])
+
+Return an immersed boundary with an irregular bottom fit to the underlying grid.
```

Comment:
the bottom isn't necessarily "irregular". The docstring should actually say what `bottom_height` is (it's an array or function that gives the height of the bottom in absolute `z` coordinates (ie not relative to the lowest level or anything like that)

Reply:
true -- thanks!